src/models/reviewDAO.js

```diff
@@ -0,0 +1,62 @@
+import prisma from '../../prisma';
+import { Prisma } from '@prisma/client';
+
+const getReviews = async () => {
+  return await prisma.$queryRaw``;
+};
+
+const createReview = async (scores, content, userId, reservationId) => {
+  await prisma.$queryRaw`
+  INSERT INTO reviews VALUES(
+              DEFAULT,
+              ${scores},
+              ${content},
+              ${userId},
+              ${reservationId},
+              DEFAULT,
+              DEFAULT,
+              DEFAULT,
+              DEFAULT
+  );
+  `;
+
+  return await prisma.$queryRaw`
+  SELECT    rev.id,
+            rev.scores,
+            rev.review_content as reviewContent,
+            rev.reservation_id as reservationId,
+            reserve.room_id as roomId,
+            r.name
+            rev.created_at as createdAt,
+            rev.updated_at as updatedAt,
+            revi.image_url as imageUrl,
+            u.nickname,
+  FROM      reviews rev
+  LEFT JOIN review_images revi
+  ON        rev.id = revi.review_id
+  LEFT JOIN 
+  ON
+  LEFT JOIN
+  ON
+  WHERE     rev.reservation_id = ${reservationId}
+  ORDER BY  rev.created_at DESC
+  LIMIT     1
+  ;`;
+};
```

Comment:
```suggestion
  return await prisma.$queryRaw`
  SELECT    rev.id,
            rev.scores,
            rev.review_content as reviewContent,
            rev.reservation_id as reservationId,
            reserve.room_id as roomId,
            r.name
            rev.created_at as createdAt,
            rev.updated_at as updatedAt,
            revi.image_url as imageUrl,
            u.nickname,
  FROM      reviews rev
  LEFT JOIN review_images revi
  ON        rev.id = revi.review_id
  LEFT JOIN 
  ON
  LEFT JOIN
  ON
  WHERE     rev.reservation_id = ${reservationId}
  ORDER BY  rev.created_at DESC
  LIMIT     1
  ;`;
};
```

간격 2 space 정도 indent